gslib/commands/rsync.py

```diff
@@ -446,6 +447,10 @@
                  what would be copied or deleted without actually doing any
                  copying/deleting.
 
+  -O             Assert ownership of objects. This is useful if uniform bucket-
```

Comment:
I see that parsing for this option was removed in the last commit so we should be removing this help text as well?